content/applications/sales/crm/acquire_leads/convert.rst

```diff
@@ -34,24 +34,26 @@ done, click :guilabel:`Save`.
 Convert a lead into an opportunity
 ==================================
 
-To convert a lead into an *opportunity*, navigate to :menuselection:`CRM --> Leads`, and click on a
-:guilabel:`Lead` from the list to open it.
+To convert a lead into an *opportunity*, navigate to :menuselection:`CRM app --> Leads`, and click
+on a lead from the list to open it.
 
-In the upper-left corner of the screen, click the :guilabel:`Convert to Opportunity` button, which
-opens a :guilabel:`Convert to opportunity` pop-up window.
+.. warning::
+   If a :guilabel:`Similar Leads` smart button appears at the top of the page for the lead, it
+   indicates a similar lead or opportunity already exists in the database. Before converting this
+   lead, click the smart button to confirm if the lead should be merged.
+
+   .. image:: convert/similar-leads-smart-button.png
+      :align: center
+      :alt: Close up of a lead with emphasis on the Similar Leads smart button.
+
+Click the :guilabel:`Convert to Opportunity` button, located at the top-left of the page.
 
 .. image:: convert/convert-leads-convert-opp-button.png
    :align: center
    :alt: Create opportunity button on a lead record.
 
-On the :guilabel:`Convert to opportunity` pop-up window, in the :guilabel:`Conversion action` field,
-select the :guilabel:`Convert to opportunity` option.
-
-.. note::
-   If a lead, or an opportunity, already exists in the database for this customer, Odoo
-   automatically suggests merging both leads/opportunities. For more information on merging leads
-   and opportunities, see the section on how to :ref:`merge leads
-   <sales/crm/acquire_leads/merge-leads>` below.
+This opens a :guilabel:`Convert to opportunity` pop-up modal. Here, in the :guilabel:`Conversion
+Action` field, select the :guilabel:`Convert to opportunity` option.
 
 Then, select a :guilabel:`Salesperson` and a :guilabel:`Sales Team` to which the opportunity should
```

Comment:
it may prove beneficial to mention whether one (or both) of these fields are required.